--- esmond-py3.orig/rpm/config_files/esmond.conf	2020-04-29 15:09:35.875879000 +0000
+++ esmond-py3/rpm/config_files/esmond.conf	2020-04-29 23:55:29.879239000 +0000
@@ -1,5 +1,6 @@
 [main]
 sql_db_engine = django.db.backends.postgresql_psycopg2
+sql_db_host = 127.0.0.1
 sql_db_name = esmond
 sql_db_user = esmond
 sql_db_password = changeit
--- esmond-py3.orig/rpm/config_files/apache-esmond.conf	2020-04-29 15:09:35.875825000 +0000
+++ esmond-py3/rpm/config_files/apache-esmond.conf	2020-04-29 23:55:29.879596000 +0000
@@ -1,16 +1,15 @@
-WSGIScriptAlias /esmond /usr/lib/esmond/esmond/wsgi.py
-WSGISocketPrefix run/wsgi
-WSGIDaemonProcess esmond python-path=/usr/lib/esmond/esmond:/usr/lib/esmond/lib/python3.6:/usr/lib/esmond/lib/python3.6/site-packages home=/usr/lib/esmond processes=10 threads=5
+WSGIScriptAlias /esmond /usr/lib/python3/dist-packages/esmond/wsgi.py
+WSGIDaemonProcess esmond processes=10 threads=5
 
 <Location /esmond>
     WSGIProcessGroup esmond
     WSGIPassAuthorization On
 </Location>
 
-<Directory /usr/lib/esmond/esmond>
+<Directory /usr/lib/python3/dist-packages/esmond>
 Header set Access-Control-Allow-Origin "*"
 <Files wsgi.py>
-SetEnv ESMOND_ROOT /usr/lib/esmond
+SetEnv ESMOND_ROOT /usr/lib/python3/dist-packages/esmond
 SetEnv ESMOND_CONF /etc/esmond/esmond.conf
 AuthType None
 <IfVersion >= 2.4>
@@ -23,8 +22,8 @@
 </Files>
 </Directory>
 
-Alias /esmond-static /usr/lib/esmond/staticfiles
-<Directory "/usr/lib/esmond/staticfiles">
+Alias /esmond-static /var/lib/esmond/staticfiles
+<Directory "/var/lib/esmond/staticfiles">
 Header set Access-Control-Allow-Origin "*"
 AllowOverride None
 <IfVersion >= 2.4>
--- esmond-py3.orig/rpm/config_files/settings.py	2020-04-29 15:09:35.876075000 +0000
+++ esmond-py3/rpm/config_files/settings.py	2020-04-29 23:55:29.879812000 +0000
@@ -63,7 +63,7 @@
 USE_I18N = True
 
 STATIC_URL = '/esmond-static/'
-STATIC_ROOT = '/usr/lib/esmond/staticfiles'
+STATIC_ROOT = '/var/lib/esmond/staticfiles'
 
 # Absolute path to the directory that holds media.
 # Example: "/home/media/media.lawrence.com/"
--- esmond-py3.orig/setup.py	2020-04-29 22:03:02.959587000 +0000
+++ esmond-py3/setup.py	2020-04-29 23:55:29.880036000 +0000
@@ -11,7 +11,10 @@
     author='Jon M. Dugan',
     author_email='jdugan@es.net',
     url='http://software.es.net/esmond/',
-    packages=['esmond', 'esmond.api', 'esmond.api.client', 'esmond.admin'],
+    packages=['esmond', 'esmond.api', 'esmond.api.client', 'esmond.admin',
+        'esmond.api.tests', 'esmond.api.tests.perfsonar', 'esmond.api.perfsonar',
+        'esmond.api.client.perfsonar', 'esmond.api.management', 'esmond.api.migrations',
+        'esmond.api.management.commands', 'esmond_client', 'test_data'],
     install_requires=[
         'Django==1.11.29',
         'web.py',
